site/source/components/SelectSimulationYear.tsx

```diff
@@ -0,0 +1,62 @@
+import { useContext } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { useDispatch, useSelector } from 'react-redux'
+import { situationSelector } from 'Selectors/simulationSelectors'
+import { Link as DesignSystemLink } from 'DesignSystem/typography/link'
+import { EngineContext } from 'Components/utils/EngineContext'
+import Banner from 'Components/Banner'
+import { updateSituation } from 'Actions/actions'
+import styled from 'styled-components'
+
+const Bold = styled.span<{ bold: boolean }>`
+	${({ bold }) => (bold ? 'font-weight: bold;' : '')}
+`
+
+export const SelectSimulationYear = () => {
+	const dispatch = useDispatch()
+	const engine = useContext(EngineContext)
+	const { t } = useTranslation()
+	const { année } = useSelector(situationSelector)
```

Comment:
```suggestion
	const année = useContext(EngineContext).evaluate('année').nodeValue
```

Aujourd'hui l'année est sauvegardée dans la situation, mais cela n'est pas voué à être tout le temps le cas. Notamment, si l'on veut éviter d'afficher les questions lors d'un changement d'année, on sera amené à changer cette implémentation. 

Il est donc plus solide de passer par une évaluation du moteur, qui est la source de vérité située au-dessus de toutes les autres dans l'application.

(ce commentaire s'applique aux autres endroits également ;) )

Reply:
J'avais fait ca au début mais en passant par `getRule` du coup j'avais toujours `2022`... Enfaite je voulais `evaluate` merci ^^

Reply:
C'est bon normalement